README.md

```diff
@@ -109,7 +109,11 @@ hv01 bond_members='eth1' bond_sub_interfaces='[{"vlanid":"100","ip":"10.0.0.11",
 hv02 bond_members='["eth1","eth2"]' bond_sub_interfaces='[{"vlanid":"100","ip":"10.0.0.12","mask":"255.255.255.0"}]'
 hv03 bond_members='["eth1","eth2"]' bond_sub_interfaces='[{"vlanid":"100","ip":"10.0.0.13","mask":"255.255.255.0"}]'
 cv01 bond_members='["eth1","eth2"]' bond_sub_interfaces='[{"vlanid":"100","ip":"10.0.0.15","mask":"255.255.255.0"}]'
+```
 
+## Sample Inventory File Part 3 - OpenStack Portion
+You can configure the OpenStack hosts and their pertinent roles (Image Host, Storage Host, DNS Host)
```

Comment:
We should add Hypervisor or VM Host at the start of the list at the end.

Reply:
Resolved